packages/opds-web-client/src/components/Book.tsx

```diff
@@ -208,7 +208,7 @@ export default class Book<P extends BookProps> extends React.Component<P, {}> {
       );
       links.push(
         <BorrowButton
-          key={this.props.book.borrowUrl}
+          key={`${this.props.book.borrowUrl}-borrow-button`}
```

Comment:
Just to fix a "key' issue. Most of the time `this.props.book.borrowUrl` is undefined.